docs/OPERATION.md

```diff
@@ -61,6 +70,25 @@ RATE_LIMIT_RPS: u32 (2)                     - Rate limit requests per second for
 SYNC_LOOKBACK_HOURS: u64 (6)                - Number of hours to look back for syncing premints from another node
 ```
 
+**Recommended Configuration for Production:**
+
+```
+DATABASE_URL=sqlite://mintpool.db
+PERSIST_STATE=true
+```
```

Comment:
Should ideally also set EXTERNAL_ADDRESS to make advertising predictable

Reply:
Mind pushing a commit with an example?